.eslintrc

```diff
@@ -9,6 +9,22 @@
         "no-alert": 0,
         "consistent-return": 0,
         "max-len": 0,
+        "import/first": 0,
```

Comment:
these new additions are here so that we don't need to fix all these lint issues from the new version of airbnb's linter which marks them as errors by default

Reply:
Before adding this, can you try using `eslint --fix` option? to see what can be automatically fixed

Reply:
@mcampa i ran through and tried `fix` after removing each of these one-by-one and was able to auto-fix a few of them.  we could create tickets or issues to go in and correct the others